src/ClientActiveRequests.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 1996-2020 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_CLIENTACTIVEREQUESTS_H
+#define SQUID_CLIENTACTIVEREQUESTS_H
+
+#include "dlink.h"
+
+extern dlink_list ClientActiveRequests;
```

Comment:
If you know the answer, please document what ClientActiveRequests are.

Reply:
Ok